test-one/src/index.js

```diff
@@ -1,14 +1,24 @@
 import { log } from "./helpers/logger";
 
 export const go = async () => {
-  // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
+    var counter;
+    var displayMessage = "";
 
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+    for (counter = 1 ; counter <= 100; counter++) {
+        if (counter % 3 && counter % 5) {
```

Comment:
Might there be benefit in moving all this logic into it's own file so that it can be reused? Can you think of any other benefit in moving into it's own file?

Reply:
Another benefit would be letting it be able to be easier to read and understand.